### DISCUSSION TITLE
Support for reply_to_message

### DISCUSSION THREAD

**Discussion by drbourbon** [def-studio/telegraph#126] [Ideas]
Hi, any plan to support `reply_to_message` Message field for incoming messages?

**Comment by fabio-ivona** [comment]
It's already supported 🔥

```php
Telegraph::message('hi!')->reply($originalMessageId);

```

**Reply by drbourbon** [reply]
Yeah this is well documented, thanks. But when handling incoming messages via a custom hook handler I need to know if that message is itself a reply to another message (`reply_to_message` field in official telegram API)

**Reply by fabio-ivona** [reply]
You're right, will implement it in a couple of days, thanks for the suggestion!

**Reply by fabio-ivona** [reply]
@drbourbon will be released in a few minutes
